qt/python/mantidqt/widgets/colorbar/colorbar.py

```diff
@@ -27,9 +27,11 @@ class ColorbarWidget(QWidget):
     colorbarChanged = Signal()  # The parent should simply redraw their canvas
     cmap_list = sorted([cmap for cmap in cm.cmap_d.keys() if not cmap.endswith('_r')])
 
-    def __init__(self, parent=None):
+    def __init__(self, parent=None, conf=None):
```

Comment:
I'm not sure this is quite the right place for the conf object in this case. The class is quite low level & could be used in a different application other than sliceviewer and this would then have the `sliceviewer/` key that is not correct.

How about instead we just pass the default scale value through to the colorbar and have the `Sliceviewer` itself do the reading/writing the `CONF`?

```python
class ColorbarWidget:
    def __init__(self, parent=None, default_scale=None):
        """
        :param default_scale: None uses linear, else either a string or tuple(string, other arguuments), e.g. tuple('Power', exponent)
        """
```

I think this is then mostly just moving the code from `set_default_scale` here to the SliceViewer.